go.mod

```diff
@@ -71,6 +71,7 @@ require (
 	github.com/uber/jaeger-lib v2.2.0+incompatible
 	go.mongodb.org/mongo-driver v1.3.2 // indirect
 	go.uber.org/atomic v1.6.0 // indirect
+	go.uber.org/goleak v1.0.0
```

Comment:
I would vote for using https://github.com/fortytw2/leaktest

1. It has less toxic community last time I checked
2. We are working on improvements that would allow running it with parallel etc.

WDYT?

Reply:
I can agree goleak is less intrusive though. leaktest does not have it :/

Still it does not show which test leaked, just overall leak, right?

Reply:
Given that goleak runs at the end of all tests, it works with tests in parallel too.

I find it great that it just works at the package level, WDYT?

For the community side, can you be more explicit?

Reply:
> I can agree goleak is less intrusive though. leaktest does not have it :/
> 
> Still it does not show which test leaked, just overall leak, right?

Yes but the README explains how to find out:

https://github.com/uber-go/goleak#determine-source-of-package-leaks